tf/modules/monitoring/sql-backup-failure-alert.tf

```
@@ -24,6 +24,12 @@ resource "google_monitoring_alert_policy" "alert_policy_sql_logical_backup_failu
     condition_absent {
       filter   = "metric.type=\"logging.googleapis.com/user/${google_logging_metric.sql_logical_backup_success.name}\" AND resource.type=\"k8s_container\""
       duration = "86400s"
+
+      aggregations {
+        cross_series_reducer = "REDUCE_COUNT"
+        per_series_aligner = "ALIGN_COUNT"
+        group_by_fields = "container_name"
```

try something like this:
```suggestion
        group_by_fields = ["container_name"]
```